core/src/style/pointStyle.cpp

```diff
@@ -61,14 +63,28 @@ void PointStyle::onBeginDrawFrame(RenderState& rs, const View& _view, Scene& _sc
 
     auto texUnit = rs.nextAvailableTextureUnit();
 
-    m_shaderProgram->setUniformi(rs, m_uTex, texUnit);
-    m_shaderProgram->setUniformMatrix4f(rs, m_uOrtho, _view.getOrthoViewportMatrix());
+    m_shaderProgram->setUniformi(rs, m_uniforms[Style::mainShaderUniformBlock].uTex, texUnit);
+    m_shaderProgram->setUniformMatrix4f(rs, m_uniforms[Style::mainShaderUniformBlock].uOrtho,
+                                        _view.getOrthoViewportMatrix());
 
     m_mesh->draw(rs, *m_shaderProgram, texUnit);
 
     m_textStyle->onBeginDrawFrame(rs, _view, _scene);
 }
 
+void PointStyle::onBeginDrawSelectionFrame(RenderState& rs, const View& _view, Scene& _scene) {
+    m_mesh->upload(rs);
+
+    Style::onBeginDrawSelectionFrame(rs, _view, _scene);
+
+    m_selectionProgram->setUniformMatrix4f(rs, m_uniforms[Style::selectionShaderUniformBlock].uOrtho,
```

Comment:
@hjanetzek  here we give a try to give a scale factor to extend the width of polylines, to fix the issue of low resolution framebuffer to select some discarded fragments (specifically for polylines) that you mentioned on slack.